gulpfile.js/client-static.js

```diff
@@ -13,13 +14,28 @@ const STATIC_PATHS = [
   '!client/src/i18n/{en,fr}/*.json',
 ];
 
+
+const CHART_JS = [
+  'node_modules/jquery/dist/jquery.min.js',
+  'node_modules/chart.js/dist/Chart.min.js',
+];
+
+
 // collects all static files from the client (BHIMA src and vendor files)
 // and moves them to the build folder, respecting folder structure
 function buildStatic() {
   return src(STATIC_PATHS)
     .pipe(dest(CLIENT_FOLDER));
 }
 
+// some report will need only chartjs and jquery
+function buildStaticChartjs() {
+  return src(CHART_JS)
```

Comment:
IF we are only using this on the server, let's just link directly into the node_modules folder from the handlebars template.

Otherwise, if we plan to use this on both the client and server, let's put it in the regular JS paths.

Reply:
Good Idea, from node_modules is very easy. I did'nt think about that